google-cloud-storage/src/main/java/com/google/cloud/storage/GrpcConversions.java

```diff
@@ -1058,7 +1065,7 @@ private Policy policyDecode(com.google.iam.v1.Policy from) {
     Policy.Builder to = Policy.newBuilder();
     ByteString etag = from.getEtag();
     if (!etag.isEmpty()) {
-      to.setEtag(etag.toStringUtf8());
+      to.setEtag(byteStringB64StringCodec.encode(etag));
```

Comment:
this is very subtle how hasn't this come up before?

Reply:
I think setIamPolicy so far has been exclusively used in json or grpc, and not yet mixed. With the conformance tests we did get mixing, because the policy is resolved in setup via json the modified via grpc.

Reply:
subtle; thanks!